libmscore/note.cpp

```diff
@@ -763,7 +763,7 @@ int Note::tpc() const
 QString Note::tpcUserName(bool explicitAccidental) const
       {
       QString pitchName = tpc2name(tpc(), NoteSpellingType::STANDARD, NoteCaseType::AUTO, explicitAccidental);
-      QString octaveName = QString::number((epitch() / 12) - 1);
+      QString octaveName = QString::number((ppitch() / 12) - 1);
```

Comment:
As far as I understand, `ppitch()` doesn't account for instrument transposition so it will always report playback pitch rather than written pitch. Woudn't it be better to just add an `ottaveCapoFret()` here like it is done elsewhere in this patch?

Reply:
I just checked again but <code>ppitch()</code> takes instruments transposition into account. It also obeys the <code>Concert Pitch</code> button. So, it does exactly what required here. That's the reason I didn't use an <code>ottavaCapoFret()</code> here.

Reply:
According to the code it does not, with this PR it is effectively the same as `_pitch + ottaveCapoFret()` where `ottaveCapoFret()` does not deal with transposition. `epitch()` function returns `_pitch - (concertPitch() ? 0 : transposition());` and honors transposition explicitly.

Reply:
You were right. In my test case, using Clarinet, everything worked fined but the test case didn't include a B or a B♭, just the two notes which didn't work in my original PR.
It is corrected.